docs/website/tds/tutorial/NcML.htm

```diff
@@ -335,10 +335,10 @@ <h2><a name="debug">Debugging NcML</a></h2>
         <li>Remove the <code>recheckEvery</code> attribute if present on the <code>&lt;scan&gt;</code> element.</li>
         <li>Make sure that the <code>&lt;scan&gt;</code> location is available on the machine you are running ToolsUI</li>
       </ol>
-      Now start up <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">ToolsUI</a>, and in the viewer tab, navigate to
+      Now start up <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">ToolsUI</a>, and in the viewer tab, navigate to
       <code>test.ncml</code> and try to open it.
     </li>
-    <li>If the dataset is <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/ncml/Aggregation.html#dynamic">dynamic</a> (files can be added or deleted),
+    <li>If the dataset is <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/ncml/Aggregation.html#dynamic">dynamic</a> (files can be added or deleted),
```

Comment:
Convert to relative links?

Reply:
Totally.

Reply:
Moved to relative (or absolute path for cross doc) links